gen/_template/response_encode.tmpl

```diff
@@ -182,6 +182,7 @@ func encode{{ $op.Name }}Response(response {{ $op.Responses.GoType }}, w http.Re
 		{{- else }}
 		writer := w
 		{{- end }}
+		defer {{ $var }}.Close()
```

Comment:
Response header encoder (above, `if $.WithHeaders`) might return an error and encoding function would return before this line.